src/types/custom.rs

```diff
@@ -88,21 +88,9 @@ pub(crate) mod test {
 
     use super::CustomType;
 
-    pub(crate) const EQ_CUST: CustomType = CustomType::new_simple(
-        SmolStr::new_inline("MyEqType"),
-        SmolStr::new_inline("MyRsrc"),
-        TypeBound::Eq,
-    );
-
     pub(crate) const COPYABLE_CUST: CustomType = CustomType::new_simple(
```

Comment:
Could replace this with FLOAT64_T?

Reply:
I guess it is actually used to test custom types later

Reply:
I'd very happily move this back, or indeed remove it, if it let me use `prelude_registry` everywhere rather than needing `test_registry` as well...but FLOAT64_T is in arithmetic which is outside prelude, right

Reply:
so it is....could be moved....

Reply:
I found at least one place calling COPYABLE_T "float" and using it as if it were such, so I replaced that one with FLOAT64_TYPE....then if I'd done one then I've now done all the others...RIP COPYABLE_T.

I've kept `test_registry` (==prelude + float_types), but could do the move yes